OJP/OJP_Locations.xsd

```diff
@@ -119,10 +119,10 @@
 	</xs:complexType>
 	<xs:complexType name="PlaceResultStructure">
 		<xs:sequence>
-			<xs:element name="Place" type="PlaceStructure"/>
+			<xs:element name="Location" type="PlaceStructure"/>
 			<xs:element name="Complete" type="xs:boolean">
 				<xs:annotation>
-					<xs:documentation>States whether the included place is complete or needs further refinement. Only complete places are fully resolved and can be used in e.g. trip requests. Incomplete places have to be refined entering them once again into a PlaceInformationRequest.</xs:documentation>
+					<xs:documentation>States whether the included location / place is complete or needs further refinement. Only complete locations / places are fully resolved and can be used in e.g. trip requests. Incomplete locations / places have to be refined entering them once again into a PlaceInformationRequest.</xs:documentation>
```

Comment:
Should be *LocationInformationRequest* in the annotation.

Reply:
done